scipy/stats/_hypotests.py

```diff
@@ -130,3 +131,133 @@ def epps_singleton_2samp(x, y, t=(0.4, 0.8)):
     p = chi2.sf(w, r)
 
     return Epps_Singleton_2sampResult(w, p)
+
+
+def poisson_etest(k1, k2, n1, n2, diff=0, alternative='two-sided'):
+    """
+    Calculate E-test for the mean difference of two samples
+
+    The test requires two samples coming from poisson distribution. This is a
```

Comment:
remove first sentence. describe the test in more detail: what is the hypothesis?

Reply:
Calculate E-test for the mean difference of two samples that follow a Poisson
    distribution from descriptive statistics

    This is a right-sided  for the null hypothesis that mean of sample one has
    difference less or equal to a specified `diff` from mean of sample two and
    two-sided test for the null hypothesis that mean of sample one is equal to
    mean of sample two added by a specified difference `diff`

what do you think for that description?